e2e/cli-e2e/tests/compare.e2e.test.ts

```diff
@@ -12,10 +12,11 @@ describe('CLI compare', () => {
         'Unstaged changes found in examples/react-todos-app, please stage or commit them to prevent E2E tests interfering',
       );
     }
-    await cleanTestFolder('tmp/e2e');
+    await cleanTestFolder('tmp/e2e/react-todos-app');
     await executeProcess({
-      command: 'code-pushup',
```

Comment:
Did not know how to fix thi. Would suggest just keep it for now. (outside of `react-todos-app` it is easier to test this in isolation)

Reply:
Using `npx @code-pushup/cli` instead of `code-pushup` is masking the root problem - local package installation isn't working, see https://github.com/code-pushup/cli/pull/810#discussion_r1751773344

BTW the tests should probably use `npx code-pushup`, the plain `code-pushup` command would only work if it's installed globally, which is an atypical setup.